src/components/login/Login.js

```diff
@@ -29,7 +28,7 @@ const Login = (props) => {
     }
 
     if (success) {
-        return <Redirect to="/" />
+        return <Redirect to={props.history.go(-2)} />
```

Comment:
Why `-2`?

Reply:
This will go back in history 2 pages, aka, the page a user is on before they logged in